src/component/management/profile/ProfileView.jsx

```diff
@@ -81,13 +92,63 @@ class ProfileView extends React.Component {
         })
     }
 
+    updateValue(e) {
+        var usr = this.state.loggedUserObj
+
+        usr[this.state.selectedPopup] = e;
+        this.setState({
+            loggedUserObj: usr,
+            isHidden: !this.state.isHidden
+        });
+        this.saveProfile();
+    }
+    updatePicture(e) {
+        var usr = this.state.loggedUserObj
+        usr.picture = e;
+        this.setState({
+            loggedUserObj: usr,
+            isHidden: !this.state.isHidden
+        });
+        this.saveProfile();
+    }
+    deletePicture() {
+      var usr = this.state.loggedUserObj
+      axios.post(Config.API_URL + AppConstants.DELETE_PICTURE, {
+          name: this.state.loggedUserObj.picture
+      })
+      .then(res => { // then print response status
+        console.log(res.statusText);
```

Comment:
Remove this console.log 
Don't keep any console.log s in front end related files.